task/acs-deploy-check/0.1/acs-deploy-check.yaml

```diff
@@ -39,7 +38,17 @@ spec:
         secretName: $(params.rox-secret-name)
         optional: true
   steps:
-    - name: rox-deploy-scan
+    - name: annotate-task
+      image: registry.redhat.io/openshift4/ose-cli:4.13@sha256:73df37794ffff7de1101016c23dc623e4990810390ebdabcbbfa065214352c7c
+      script: |
+        #!/usr/bin/env bash
+        echo "acs-image-scan $(context.taskRun.name)"
+        oc annotate taskrun $(context.taskRun.name) task.results.format=application/json
+        oc annotate taskrun $(context.taskRun.name) task.results.type=roxctl-deployment-check
+        oc annotate taskrun $(context.taskRun.name) task.results.container=step-report
+        oc annotate taskrun $(context.taskRun.name) task.output.location=logs
```

Comment:
This is needed because the annotations from the Task don't always get applied to the TaskRun?

Reply:
Yes, due to a bug in Tekton, they just removed.